release_collection.py

```diff
@@ -340,6 +406,15 @@ def update_collection(args, galaxy, coll_rel):
                 check_versions_updated(
                     cur_ref, coll_rel[rolename]["ref"], versions_updated
                 )
+            # If a version update is detected, retrieve the new changelogs from CHANGELOG.md
+            if any(versions_updated):
```

Comment:
You can't use `versions_updated` for individual roles - `versions_updated` keeps track of the semantic version changes to **all** roles (listed in `args.include`) - unfortunately `check_versions_updated` doesn't tell you if the version for the given role has changed - you'll have to modify it to do that (perhaps figure out a way to combine that with `comp_versions`